ui/src/components/EventConfiguration/eventConfigXmlValidator.ts

```diff
@@ -21,16 +22,22 @@ export const validateEventConfigFile = async (file: File) => {
     if (validationErrors.length === 0) {
       const parser = new DOMParser()
       const xmlDoc = parser.parseFromString(text, 'application/xml')
+      const result = XMLValidator.validate(text)
```

Comment:
Regarding this line above:

`if (!file.name.endsWith('.events.xml') && !file.name.includes('event')) {`

Is it a requirement to have files named this way? I think we are getting rid of the idea of a file since everything is in the database. Maybe we can check for an `.xml` extension, but not sure if we need the rest.

If the back end requires this naming, then that is ok for now.